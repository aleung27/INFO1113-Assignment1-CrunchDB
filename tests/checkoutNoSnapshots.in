CHECKOUT 1
CHECKOUT 0
SET B 8
SNAPSHOT
CHECKOUT 0
DROP 1
CHECKOUT 1
BYE
