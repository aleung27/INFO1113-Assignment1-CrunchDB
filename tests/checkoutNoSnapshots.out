> no such snapshot

> no such snapshot

> ok

> saved as snapshot 1

> no such snapshot

> ok

> no such snapshot

> bye
